lib/BaseDisplayObject.jsx

```diff
@@ -3,7 +3,9 @@
 import React from 'react';
 
 export default class BaseDisplayObject extends React.Component {
-
+  
+  state = {};
```

Comment:
It would if you'd use `createClass`. With ES7 (stage 1) you also can assign class properties directly without calling methods. See https://github.com/jeffmo/es-class-fields-and-static-properties.